com.teamscale.polarion.plugin/src/com/teamscale/polarion/plugin/utils/Utils.java

```diff
@@ -0,0 +1,314 @@
+package com.teamscale.polarion.plugin.utils;
+
+import com.polarion.alm.projects.model.IUser;
+import com.polarion.alm.tracker.model.IApprovalStruct;
+import com.polarion.alm.tracker.model.IAttachment;
+import com.polarion.alm.tracker.model.ICategory;
+import com.polarion.alm.tracker.model.IComment;
+import com.polarion.alm.tracker.model.IHyperlinkStruct;
+import com.polarion.alm.tracker.model.ILinkedWorkItemStruct;
+import com.polarion.alm.tracker.model.IModule;
+import com.polarion.alm.tracker.model.ITestSteps;
+import com.polarion.alm.tracker.model.IWorkItem;
+import com.polarion.core.util.types.duration.DurationTime;
+import com.polarion.platform.persistence.IEnumOption;
+import com.polarion.platform.persistence.model.IPObject;
+import com.polarion.subterra.base.location.ILocation;
+import com.teamscale.polarion.plugin.model.LinkedWorkItem;
+import com.teamscale.polarion.plugin.model.WorkItemForJson;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Utils {
+
+  public static String LINKED_WORK_ITEMS_FIELD_NAME = "linkedWorkItems";
+
+  public enum UpdateType {
```

Comment:
This enum should probably reside in the model package

Reply:
Done